src/client.ts

```diff
@@ -12,6 +12,21 @@ export function makeClient(config: LanguageConfig): vscode.Disposable {
   const { languageId, serverCommand } = config
   if (!languageId || !serverCommand) return vscode.Disposable.from()
 
+  const serverExecutable = {
```

Comment:
The LSP client library will manage the child process for you if you just give it a process spec.

This is how the official Haskell extension launches HLS.